docs/api.md

```diff
@@ -362,7 +362,26 @@ particular user.
 
 - `client_id`: The id returned from client registration.
 - `client_secret`: The secret returned from client registration.
-- `code`: A string that was received from the [authorization][] endpoint.
+- `ttl`: (optional) Seconds that this access_token should be valid.
```

Comment:
nit: `Seconds that this access_token should be valid.` -> 

`The amount of time the access_token is valid for in seconds` or 
`The auth_token expiration time in seconds`